Sources/DeepLearning/Operators.swift

```diff
@@ -26,3 +26,326 @@ public func round<Scalar: BinaryFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Sc
 public func identity<Scalar>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
     return x
 }
+
+//===----------------------------------------------------------------------===//
```

Comment:
Could you extend these to be 100 columns long?

Reply:
done